### ISSUE TITLE
Calculating ELECTRA infer FLOPs

### ISSUE THREAD

**Issue by asharma20** [google-research/electra#65]
I'm trying to calculate ELECTRA inference FLOPs since `flops_computation.py` doesn't include this. Would the calculation for this be similar to how the other transformer infer flops are called, where the hidden size and layer number is taken from the discriminator? 
`electra_infer_flops = TransformerHparams(h_d, l_d, s=s, e=e, output_frac=output_frac).get_infer_flops()`

Are there any other changes I need to make to the code? From what I understand, ELECTRA uses the first element for `self.pooled_output` instead of calculating it using dense like BERT does, so do I have to remove everything except the `logits` key in the `get_binary_classification_flops()` function?
https://github.com/google-research/electra/blob/79111328070e491b287c307906701ebc61091eb2/flops_computation.py#L121-L123

**Comment by clarkkev** [comment]
If I understand right you are asking about ELECTRA's infer FLOPs when used as a binary sentence classifier? In that case you can call `get_infer_flops()` with `output_frac=0.0` (since there is no output softmax when doing classification tasks). To be 100% exact, you would also need to add some flops for the final softmax layer, but that compute is pretty tiny compared to the compute for running the transformer. `get_binary_classification_flops()` isn't the right thing to use because it is assuming a binary classification task on every token as ELECTRA does during pre-training, hence the multiplication by `self.s`.
